dynamo/plot/utils.py

```diff
@@ -274,26 +282,28 @@ def _matplotlib_points(
 
         if frontier:
             main_debug("drawing frontier")
-            rasterized = kwargs["rasterized"] if "rasterized" in kwargs.keys() else None
-            ax.scatter(
-                points[:, 0],
-                points[:, 1],
-                kwargs["s"] * 2,
-                "0.0",
+            _scatter_projection(
+                ax,
+                points,
+                projection,
+                s=kwargs["s"] * 2,
+                c="0.0",
                 lw=2,
                 rasterized=rasterized,
             )
-            ax.scatter(
-                points[:, 0],
-                points[:, 1],
-                kwargs["s"] * 2,
-                "1.0",
+            _scatter_projection(
+                ax,
+                points,
+                projection,
+                s=kwargs["s"] * 2,
+                c="1.0",
                 lw=0,
                 rasterized=rasterized,
             )
-            ax.scatter(
-                points[:, 0],
-                points[:, 1],
+            _scatter_projection(
```

Comment:
should we also add `rasterized=rasterized,` here?

Reply:
The old code does not have rasterized=rasterized here. I am not sure if I add some argument to ~2000 lines of code function "scatters" can cause any problem.

Reply:
rasterized=True means rasterize the data points which will save a lot space when plotting the scatter. 

This argument won't affect any other part of the code. If you are not sure, just test it out.